server/routes/metrics.js

```diff
@@ -1,21 +1,47 @@
 const { sendBatch, clientEvent } = require('../amplitude');
+const statsdClient = require('statsd-client');
+const config = require('../config');
+
+async function sendToAmplitude(data, req) {
+  const deltaT = Date.now() - data.now;
+  const events = data.events.map(e =>
+    clientEvent(
+      e,
+      req.ua,
+      data.lang,
+      data.session_id + deltaT,
+      deltaT,
+      data.platform,
+      req.ip
+    )
+  );
+  return sendBatch(events);
+}
+
+function sendToStatsd(data) {
+  let client = new statsdClient({
+    host: config.statsd_host,
+    port: config.statsd_port,
+    prefix: config.statsd_prefix
+  });
+
+  data.events.forEach(function(e) {
+    let metric_name = e.event_type;
+    if (e.event_properties.status) {
+      metric_name += `_${e.event_properties.status}`;
```

Comment:
Appending the status allows us to differentiate between events of the same type with different results. e.g:
`cancelledUpload` -> `client_upload_cancel`
`completedUpload` -> `client_upload_ok`